res/skins/Shade/mixer_panel.xml

```diff
@@ -558,6 +558,76 @@
                 </Connection>
               </PushButton>
 
+              <!--
+              **********************************************
+              Visual- Recording State
+              **********************************************
+              -->
+              <!--Button as visual indicator for recording state-->
+			  <PushButton>
+                <NumberStates>3</NumberStates>
+                <State>
+                  <Number>0</Number>
+                  <Pressed>skin:/btn/btn_record.png</Pressed>
+                  <Unpressed>skin:/btn/btn_record.png</Unpressed>
+                </State>
+                  <State>
+                  <Number>1</Number>
+                  <Pressed>skin:/btn/btn_record_overdown.png</Pressed>
+                  <Unpressed>skin:/btn/btn_record_overdown.png</Unpressed>
+                </State>
+                <State>
+                  <Number>2</Number>
+                  <Pressed>skin:/btn/btn_record_over.png</Pressed>
+                  <Unpressed>skin:/btn/btn_record_over.png</Unpressed>
+                </State>
+                <Pos>214,18</Pos>
+                <Connection>
+                  <ConfigKey>[Recording],status</ConfigKey>
+                </Connection>
+              </PushButton>
+
+               <!--
+              **********************************************
+              Text- Recording Duration
+              **********************************************
+              -->
+              <RecordingDuration>
+				<Style>
+                  QLabel {
+                    font-size: 9px;
+                    background-color: transparent;
+                    color: #191F24;
+                    text-align:center;
+                  }
+                </Style>
+				<ObjectName>RecDuration</ObjectName>
+				<TooltipId>recording_duration</TooltipId>
+				<Pos>216,18</Pos>
+				<Size>26f,14f</Size>
+				<InactiveText>REC</InactiveText>
+				<Alignment>center</Alignment>
+		      </RecordingDuration>
+
+		      <!--
+              **********************************************
+              Button- Toggle Recording
+              **********************************************
+              -->
+              <!--invisible Rec button-->
+		      <PushButton>
+				<TooltipId>toggle_recording</TooltipId>
+                <Pos>214,18</Pos>
+                <Connection>
+                  <ConfigKey>[Recording],toggle_recording</ConfigKey>
+                  <EmitOnDownPress>true</EmitOnDownPress>
+                  <ButtonState>LeftButton</ButtonState>
+                </Connection>
+                <Connection>
+                  <ConfigKey>[Recording],toggle_recording</ConfigKey>
+                </Connection>
+              </PushButton>
```

Comment:
for some reason this button is 100x30px big, so it is much bigger that the graphic below and spans into the mixer region.
either set the size directly with `<Size>` or use an empty PNG with that size.